inc/views/base/calendar/calendar.twig

```diff
@@ -0,0 +1,105 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.calendar }}</title>
+{% endblock head %}
+
+{% block body %}
+    <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+        <thead>
+            <tr>
+                <td class="thead" colspan="8">
+                    <div class="float_right">
+                        {% if calendar.prev_link %}
+                            <a href="{{ calendar.prev_link|raw }}">&laquo; {{ calendar.prev_month_name }} {{ calendar.prev_month_year }}</a>
+                        {% endif %}
+                        {% if calendar.sep == true %}
```

Comment:
Could simplify this to `{% if calendar.sep %}`, since you already use the short form for `next_link` below (unless this can be a string or some other type or something...).